src/foam/u2/dialog/ApplicationPopup.js

```diff
@@ -389,18 +394,27 @@ foam.CLASS({
                 .tag(dynamicFooter);
             }))
           .end()
-          .callIf((this.footerString || this.includeSupport ), function() {
+          .callIf((this.footerStrings || this.includeSupport ), function() {
             this.start()
               .addClasses([self.myClass('footer'), 'p-legal-light'])
               // empty space
               .start().addClass(self.myClass('footer-left'))
               .end()
               // link
               .start().addClass(self.myClass('footer-center'))
-                .start(self.footerLink ? 'a' : '')
-                  .show(self.footerString$)
-                  .enableClass(this.myClass('footer-link'), this.footerLink$)
-                  .add(self.footerString$)
+                .start(self.footerLink ? 'a' : 'p')
+                  .show(self.footerStrings$)
+                  .enableClass(self.myClass('footer-link'), self.footerLink$)
+                  .forEach(self.footerStrings, function(str) {
```

Comment:
Can we just use htmlView here?

Reply:
Will allow for any sort of formatted view we might want

Reply:
https://github.com/kgrgreer/foam3/blob/451768a68acfbebc1167cfe1cb0acdcd8632c44a/src/foam/u2/wizard/views/FocusWizardView.js#L87-L93